RoomBookingSystem/database/migrations/2025_04_23_062905_create_rooms_groups_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('rooms_groups', function (Blueprint $table) {
```

Comment:
Laravel has some expectations when it comes to table naming, I suggest following those.
I found this article explaining table names in laravel <https://accreditly.io/articles/naming-conventions-for-laravel-database-tables>
```suggestion
        Schema::create('group_room', function (Blueprint $table) {
```